core/api/full_node/make.cpp

```diff
@@ -42,15 +42,11 @@
 #include "vm/state/impl/state_tree_impl.hpp"
 #include "vm/toolchain/toolchain.hpp"
 
-#define MOVE(x)  \
-  x {            \
-    std::move(x) \
-  }
+#define MOVE(x) \
+  x { std::move(x) }
 
-#define FWD(x)                   \
-  x {                            \
-    std::forward<decltype(x)>(x) \
-  }
+#define FWD(x) \
+  x { std::forward<decltype(x)>(x) }
```

Comment:
Is it clang-format 11?  
My clang-format 11.0.0 formats as old (red), so it will revert this change.